crates/oxc_linter/src/rules/oxc/only_used_in_recursion.rs

```diff
@@ -371,8 +350,8 @@ fn is_property_only_used_in_recursion_jsx(
             return false;
         };
 
-        let Some(attr) = ctx.nodes().ancestors(may_jsx_expr_container.id()).find_map(|node| {
-            if let AstKind::JSXAttributeItem(attr) = ctx.nodes().get_node(node).kind() {
+        let Some(attr) = ctx.nodes().iter_parents(may_jsx_expr_container.id()).find_map(|node| {
```

Comment:
you will that in `main` this is called `ancestors` again

Reply:
Thank you! Resolved in: aa4878ccdab37424f155d7bfc347be4ed2ff1d4e